src/BehaviorsSDKManaged/Microsoft.Xaml.Interactions/Core/ChangePropertyAction.cs

```diff
@@ -4,9 +4,11 @@ namespace Microsoft.Xaml.Interactions.Core
 {
     using System;
     using System.Globalization;
+    using System.Linq;
     using System.Reflection;
-    using Windows.UI.Xaml;
     using Interactivity;
+    using Uno.Extensions.Specialized;
```

Comment:
Why do you need this using ? It's not available on UAP.

Reply:
I don't remember how did that appear there.
Because I was not managed to build the project with `uap10` target in a list I've just deleted it and tested without.
If there is a manual how to make it build with `uap10` target in a list I would be glad to know.